Updated final version

yashakaushal opened the pull request:
Fixed a bug in saving hops_list 
Added failure cases from user input
Added tycho-2 catalog, need database first to uncomment and use.

yashakaushal committed:
Fixed a bug in saving hops_list and added failure cases from user input 

Added tycho-2 catalog, need database first to uncomment and use.

pandaliza commented on Final Project/Editor Interface/editor_interface_final.py:
```diff
@@ -1,24 +1,21 @@
+ms = pd.read_csv('/Users/yashakaushal/Documents/summer_project/interface/messier_objects.csv')
+cb = pd.read_csv('/Users/yashakaushal/Documents/summer_project/interface/constellation_borders.csv')
+ty = pd.read_csv('/Users/yashakaushal/Documents/summer_project/interface/tycho_1.csv')
 
```
So, if we use these database importing code, it won't work in someone else local machines. The editor would have change these manually. Can we keep these as what it is right now? It'll work in generally anyone's computer.

yashakaushal replied:
Right. If we keep all these .csv files in a folder in the repo, we can only mention that path and the user while cloning need not change anything.

yashakaushal replied:
Fixed this by defaulting the file path to Database folder.

pandaliza commented on Final Project/Editor Interface/editor_interface_final.py:
```diff
@@ -266,142 +284,132 @@ class ClickInfo(plugins.PluginBase):
-def func2(ra, dec, mag, fov):
-    # sorting objects under the user input of limiting magnitude;
-    # also duplicating the the objects and transforming them so they are repeated to the left of y-axis
-    mag_ng = ng[(ng["V"] <= mag)]
```
Can you check in the final database that is provided by Shyadari? I think it uses `V` instead of `mag`, that's why I think I changed this initially.

yashakaushal replied:
That is correct. I checked in the new catalog of NGC.csv, the column names are changed. So I'll need to modify RA/DECs as well. Will make another commit.

yashakaushal replied:
In the new commit, I have fixed all the column discrepancies. It should work now.

yashakaushal committed:
Changed paths of database files 

All database .csv files should be in 'Database/' folder 
Changing column names to be done.

yashakaushal committed:
File paths changed to default and column names fixed 

Column names updated as per new Database. 
File paths directed to Database folder.

yashakaushal committed:
Paths fixed 

All .csv files should be in 'Database/' directory after the editor runs the script given by Sahyadri to download the files.
